srt.go

```diff
@@ -34,17 +34,30 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 	// Scan
 	var line string
 	var lineNum int
+
+	// For the Index Counter
+	saveIndex := true
+	var index int
+	var convErr error
+
 	var s = &Item{}
 	for scanner.Scan() {
 		// Fetch line
 		line = strings.TrimSpace(scanner.Text())
+
+		// Fetch Index
```

Comment:
I'd rather not extract the index this way. See below

Reply:
@asticode Fixed. :+1: